addons/binding/org.openhab.binding.regoheatpump/src/main/java/org/openhab/binding/regoheatpump/RegoHeatPumpBindingConstants.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.regoheatpump;
+
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link RegoHeatPumpBinding} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Boris Krivonog - Initial contribution
+ */
+public class RegoHeatPumpBindingConstants {
+
+    public static final String BINDING_ID = "regoheatpump";
+
+    // List of all Thing Type UIDs
+    public final static ThingTypeUID THING_TYPE_IP_REGO6XX = new ThingTypeUID(BINDING_ID, "ipRego6xx");
```

Comment:
Non-preferred order of modifiers (It was like this in a template, which has been changed recently) correct order is `public static final` See also: http://stackoverflow.com/questions/16731240/what-is-a-reasonable-order-of-java-modifiers-abstract-final-public-static-e#16732059